docs/generated/errors-list.md

```diff
@@ -237,6 +237,8 @@ The `galasactl` tool can generate the following errors:
 - GAL1238E: Failed to get streams. Unexpected http status code {} received from the server. Error details from the server are not in a valid json format. Cause: '{}'
 - GAL1239E: Failed to get streams. Unexpected http status code {} received from the server. Error details from the server are: '{}'
 - GAL1240E: Failed to get streams. Unexpected http status code {} received from the server. Error details from the server are not in the json format.
+- GAL1241E: The test stream could not be deleted by name because it was not found by the Galasa service. Try listing streams using 'galasactl streams get' to identify the one you wish to delete
+- GAL1242E: Failed to test stream from database by stream name.
```

Comment:
Missing "delete", also "database" could be clarified. Maybe reword this message to something like "Failed to delete test stream with the given name from the Galasa service"?